blues/ncmc.py

```diff
@@ -1,3 +1,5 @@
+from __future__ import print_function
```

Comment:
This is something that works regardless? i.e. we don't have to check it's in python < 3?